src/notification/NotificationManager.tsx

```diff
@@ -82,9 +82,11 @@ export default class NotificationManager {
           case UserNotificationType.END_OF_SESSION:
           case UserNotificationType.SESSION_STARTED:
             break;
-          // Force display notif
+          // Display notif
           default:
-            await firebase.notifications().displayNotification(notification);
+            if (notification.data.notificationType in UserNotificationType) {
```

Comment:
Did you test it with a real notification?

Reply:
No but the code is pretty straightforward and the matching of notification.data.notificationType with UserNotificationType is known to work.

Reply:
You know what I think about your straightforward coding ;-)
Not tested => not merged
Moreover:
console.log('SessionStarted' in UserNotificationType);
=> return false instead of true.
Could you check?

Reply:
> You know what I think about your straightforward coding ;-)
> Not tested => not merged

The mobile application have a beta test phase for that purpose.

Reply:
> The mobile application have a beta test phase for that purpose.

No, you test your code or you'll have a problem with me ;-)